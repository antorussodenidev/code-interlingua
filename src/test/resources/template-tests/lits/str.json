{
    "type": {
        "nodeType": "ClassOrInterfaceType",
        "code": "java.lang.String"
    },
    "value": "foo\nbar"
}
