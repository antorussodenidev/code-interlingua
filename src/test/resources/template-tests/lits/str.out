package foo.bar;

class C {
  static final java.lang.String X = "foo\nbar";

  public C() {}
}
